client/hooks/useFocusOnMount.js

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
Just need to add a quick comment here noting that this can be removed in the future:

```
/**
 * This hook was directly copied from https://github.com/WordPress/gutenberg/blob/master/packages/compose/src/hooks/use-focus-on-mount/index.js
 * to avoid its absence in older versions of WordPress.
 *
 * This can be removed once the minimum supported version of WordPress includes this hook.
 */
```

Reply:
The PR is ready